src/Parser.js

```diff
@@ -11,15 +11,16 @@ class Parser
 
         /* Build the board */
         const boardTemplate = config_data.board;
+        const indexRenderer = IndexRenderers.getRenderer(config_data.indexRenderer);
```

Comment:
What do you do if you get bad input?
(which reminds me, I don't think I have error checking anywhere else, but it's a valid question I need to fix throughout the code base anyway)